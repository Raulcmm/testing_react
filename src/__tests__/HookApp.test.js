import { shallow } from "enzyme";
import React from "react";

import HookApp from "../HookApp";

describe("Tests to <HookApp/>", () => {
	test("match", () => {
		const wrapper = shallow(<HookApp />);
		expect(wrapper).toMatchSnapshot();
	});
});
