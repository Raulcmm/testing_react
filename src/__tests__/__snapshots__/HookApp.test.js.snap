// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests to <HookApp/> match 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
